packages/web-data/src/hooks/useIsChainType.ts

```diff
@@ -0,0 +1,15 @@
+import { useEVMContext } from '../context';
+
+export function useIsChainType() {
```

Comment:
is it missing `isMantle`?

Reply:
Yeah, guess we never ended up using this hook :P - will add